lm_eval/api/task.py

```diff
@@ -88,6 +88,7 @@ class TaskConfig(dict):
     filter_list: Union[str, list] = None
     should_decontaminate: bool = False
     doc_to_decontamination_query: str = None
+    weight_by_size: bool = False
```

Comment:
I think I mentioned this on the recursive benchmarks PR, but I think that `weight_by_size` should definitely be configured in yamls for benchmarks/groups, not in each subtask. Subtasks will be in several groups that might compete for this attribute